charts/prometheus-blackbox-exporter/templates/servicemonitor.yaml

```diff
@@ -40,8 +40,8 @@ spec:
       - targetLabel: {{ $targetLabel | quote }}
         replacement: {{ $replacement | quote }}
         {{- end }}
-{{- if concat .additionalRelabeling $.Values.serviceMonitor.defaults.additionalRelabeling }}
-{{ toYaml (concat .additionalRelabeling $.Values.serviceMonitor.defaults.additionalRelabeling) | indent 6 }}
+{{- if concat (.additionalRelabeling | default list) $.Values.serviceMonitor.defaults.additionalRelabeling }}
```

Comment:
Why not just?

```suggestion
{{- if $.Values.serviceMonitor.defaults.additionalRelabeling }}
```

Reply:
For cases where a user has an `additionalRelabeling` action for a single servicemonitor but not a default one.